internal/services/compute/shared_image_version_resource.go

```diff
@@ -177,6 +184,13 @@ func resourceSharedImageVersion() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"replicated_region_deletion_enabled": {
```

Comment:
tbh this'd be clearer/closer to the original name whilst still consistent as:

```suggestion
			"deletion_of_replicated_locations_enabled": {
```

Reply:
code updated. Thanks.